db/migrate/20231027144605_add_unit_amount_to_fees.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class AddUnitAmountToFees < ActiveRecord::Migration[7.0]
+  def up
+    change_table :fees, bulk: true do |t|
+      t.change :unit_amount_cents, :bigint, null: false, default: 0
+    end
+    add_column :fees, :unit_amount, :decimal, precision: 30, scale: 5, null: false, default: '0.0'
+
+    execute <<-SQL
+      UPDATE fees
+      SET unit_amount = unit_amount_cents / 100.0
```

Comment:
This should be adapted from the currency. From example if the currency is `JPY`, `unit_amount_cents` == `unit_amount`